src/interface/src/app/login/login.component.spec.ts

```diff
@@ -2,20 +2,19 @@ import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { Router } from '@angular/router';
+import { By } from '@angular/platform-browser';
+import { MatDialog, MatDialogModule } from '@angular/material/dialog';
```

Comment:
Safe to remove this and other MatDialog references here

Reply:
Done.